corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -1,3 +1,23 @@
+"""
+EntriesContributor
+------------------
+
+This is the largest and most complex of the suite sections, responsible for generating an ``<entry>``
+element for each form, including the datums required for form entry. The ``EntriesHelper``, which does all of the
+heavy lifting here, is imported into other places in HQ that need to know what datums a form requires,
+such as the session schema generator for form builder and the UI for form linking.
+
+Each form's required datums are generated separately in ``corehq.apps.app_manager.xform`` so they can be included in the
+form's casexml. The datums in the casexml need to match the datums in the suite ``<entry>`` blocks, or the form
```

Comment:
I found this sentence confusing. The `xform` module actually depends on `EntriesHelper` for knowing the datum IDs: https://github.com/dimagi/commcare-hq/blob/801acc530b4d6dd55a12f2de208e71408a9bd506/corehq/apps/app_manager/xform.py#L118-L129

Reply:
Hmm, I see, yeah that isn't accurate. I was thinking of issues that @esoergel identified during https://github.com/dimagi/commcare-hq/pull/31613, there was something about casexml datums not matching suite datums. Maybe there's renaming logic that's different in the xform vs the suite? I don't think I remember well enough to articulate it, so I just removed these lines. 90d07368825